Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -2560,6 +2560,11 @@ class WebCore::TransformOperations {
     WebCore::Pattern::Parameters parameters();
 }
 
+[Nested, AdditionalEncoder=StreamConnectionEncoder] struct WebCore::SourceBrush::Brush::LogicalGradient {
+    std::variant<Ref<WebCore::Gradient>, WebCore::RenderingResourceIdentifier> gradient;
```

Comment:
I'm pretty sure we used to be sending a std::optional<WebCore::RenderingResourceIdentifier>.  Is there a conscious reason why that changed?

Reply:
The type is:
std::variant<Ref<Gradient>, RenderingResourceIdentifier>

I don't see where there's an optional<RenderingResourceIdentifier> either before or after.

That said, looking more closely *did* reveal a weird asymmetry in the hand written encoder/decoder. I've asked the original author about that on Slack.